document-store/src/main/java/org/hypertrace/core/documentstore/expression/operators/FunctionOperator.java

```diff
@@ -4,7 +4,8 @@ public enum FunctionOperator {
   // Unary operations
   ABS,
   FLOOR,
-  LENGTH,
+  LENGTH, // This operator returns the length of an array,
+  // use along with DISTINCT of aggregation operator
```

Comment:
Nit: `LENGTH` can also be used independently of `DISTINCT` if the stored value is a JSON array.